src/closure.ts

```diff
@@ -20,8 +20,8 @@ const closureToJS = (value: Closure, context: Context, klass: string) => {
   })
   Object.setPrototypeOf(DummyClass, () => undefined)
   Object.defineProperty(DummyClass, 'Inherits', {
-    value: (Parent: Value) => {
```

Comment:
This is for OOP. Doesn't make sense for it to be renamed to tail.

Reply:
I agree. @SwampertX please change this back to Parent.